dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/runner/GlobalTaskDispatchWaitingQueueLooper.java

```diff
@@ -91,6 +88,7 @@ public void close() throws Exception {
         if (RUNNING_FLAG.compareAndSet(true, false)) {
             log.info("GlobalTaskDispatchWaitingQueueLooper stopping...");
             log.info("GlobalTaskDispatchWaitingQueueLooper stopped...");
+            workerGroupTaskDispatcherManager.close();
```

Comment:
Should before `log.info("GlobalTaskDispatchWaitingQueueLooper stopped...");`

Reply:
My mistake led to the meaningless log. I will make the changes as suggested.